File: lua/1006.lua
local A = io.read("*n");
local B = io.read("*n");
local C = io.read("*n");

A = A * 2.0
B = B * 3.0
C = C * 5.0

local MEDIA = (A+B+C)/10.0

print("MEDIA = " .. string.format("%0.1f", MEDIA))

File: c/1006.c
#include <stdio.h>

int main()
{
    double A, B, C;
    scanf("%lf", &A);
    scanf("%lf", &B);
    scanf("%lf", &C);

    A = A * 2.0;
    B = B * 3.0;
    C = C * 5.0;

    double MEDIA = (A+B+C)/10.0;

    printf("MEDIA = %0.1lf\n", MEDIA);

	return 0;
}

